<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shapeForExplanation</class>
 <widget class="QWidget" name="shapeForExplanation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>749</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1149</width>
    <height>749</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Andalus</family>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Work Report</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 108);</string>
  </property>
  <widget class="QPushButton" name="managment">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andalus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>5</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Schedular">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andalus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FCFS">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>550</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andalus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HRRN">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>550</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andalus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SPN">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>550</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andalus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SRT">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>550</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andalus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RR">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>550</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andalus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="suggest">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>550</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andalus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="process">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>220</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andalus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>5</width>
     <height>155</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>510</y>
     <width>160</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>510</y>
     <width>5</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>5</width>
     <height>145</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>500</y>
     <width>230</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>500</y>
     <width>5</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>370</y>
     <width>5</width>
     <height>135</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_12">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>520</y>
     <width>60</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>520</y>
     <width>5</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_14">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>370</y>
     <width>5</width>
     <height>155</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_15">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>520</y>
     <width>60</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_16">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>520</y>
     <width>5</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_17">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>370</y>
     <width>5</width>
     <height>145</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_18">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>5</width>
     <height>135</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_19">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>510</y>
     <width>160</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_20">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>500</y>
     <width>240</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_21">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>510</y>
     <width>5</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_22">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>500</y>
     <width>5</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_23">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>250</y>
     <width>150</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_24">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>140</y>
     <width>5</width>
     <height>195</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_25">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>140</y>
     <width>60</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_26">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>110</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_27">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>240</y>
     <width>5</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_28">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>240</y>
     <width>5</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_29">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>290</y>
     <width>40</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_30">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>280</y>
     <width>40</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="LinkListRR">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>660</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="linkListSRT">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>660</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="Line" name="line_31">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>640</y>
     <width>80</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_32">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>610</y>
     <width>5</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_33">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>640</y>
     <width>5</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_34">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>610</y>
     <width>5</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_35">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>640</y>
     <width>80</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_36">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>640</y>
     <width>5</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>work report</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andalus</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#next:hover{
	background-color: rgb(209, 69, 0);
};
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QLabel" name="line1">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>110</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>line_1</string>
   </property>
  </widget>
  <widget class="QLabel" name="line2">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>150</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>line_2</string>
   </property>
  </widget>
  <widget class="QLabel" name="line3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>line_3</string>
   </property>
  </widget>
  <widget class="Line" name="line_37">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>50</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_38">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>50</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_39">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>50</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_40">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>5</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_41">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>5</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="line3_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>line_3</string>
   </property>
  </widget>
  <widget class="QLabel" name="line3_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>line_3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
